### DISCUSSION TITLE
Addition of word in Citrinet Model

### DISCUSSION THREAD

**Discussion by sharaddobariya05** [NVIDIA-NeMo/NeMo#4899] [General]
stt_en_citrinet_512 and stt_en_citrinet_1024 both are trained on large English speech data, is there any way we can add few more words such as name(Example Hanuman or Ganesha), pronunciation of these words might be different depends upon the language , if we have enough speech data to add these words for improvement its transcription accuracy.

In short, I want to add few more words in dictionary of this model.

**Comment by titu1994** [comment]
If you export and run inference via NVIDIA Riva, they have word boosting which would boost certain words and spellings the model was not trained on. Nemo currently does not support that. 

You can also use adapters with very few update steps to tune your model to recognise certain words better. Or fine-tune with very low wer and for a fee steps only v
